src/frontend/packages/core/src/shared/components/cards/card-cf-org-user-details/card-cf-org-user-details.component.spec.ts

```diff
@@ -24,7 +25,13 @@ describe('CardCfOrgUserDetailsComponent', () => {
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [CardCfOrgUserDetailsComponent, ...MetadataCardTestComponents, CapitalizeFirstPipe, BooleanIndicatorComponent],
+      declarations: [
+        CardCfOrgUserDetailsComponent,
+        MetadataItemComponent,
+        MetadataCardTestComponents,
```

Comment:
I searched `MetadataCardTestComponents` among the other files and just this one was using the spread operator, so I removed it and it worked. But I'll try removing it since you said it's probably not necessary.

Reply:
Just realized that we are not using the meta card components anymore, so it makes sense to remove it.